docker-compose.yml

```
@@ -0,0 +1,10 @@
+version: '3.7'
+services:
+  redis:
+    image: redis:alpine
+    command: redis-server --port 6379
+    container_name: redis
+    labels:
+      - "name=redis"
+    ports:
+      - 6379:6379
```

```suggestion
      - 6379:6379
      
```

마지막 라인에 개행을 추가하는 것이 POSIX 표준입니다 :)

---

앗 변경하겠습니다.